case-lib/lib.sh

```diff
@@ -293,6 +293,25 @@ find_ldc_file()
     printf '%s' "$ldcFile"
 }
 
+func_cavstool_etrace_collect()
+{
+    local clogopt="--log-only --verbose"
+    local clogfile=$LOG_ROOT/etrace.txt
+
+    if [ -z "$CAVSTOOL" ]; then
+        CAVSTOOL=$(command -v cavstool.py) || {
+            dlogw 'No cavstool.py found in PATH'
+            return 1
+        }
+    fi
+
+    local cavstoolCmd=("$CAVSTOOL" "$clogopt")
```

Comment:
```suggestion
    local cavstoolCmd=("$CAVSTOOL"  --log-only --verbose)
```

Otherwise this tries to load a firmware file named `'--log-only --verbose'`

I just wasted a couple of hours on this... already asked to inline this pointless variable before (when it was a single option and working)

Reply:
OK, I realize only I see this because I disable the sof-test `sudo` function, I'm using the regular `sudo` command.

Should still be fixed.

Reply:
Please help review and merge this `sudo()` fix first:
- #943